services/QuillLMS/config/initializers/sidekiq.rb

```diff
@@ -37,4 +37,10 @@ module SidekiqQueue
   # Giving them their own queue lets us turn migrations entirely off rather
   # than risking normal workers getting tied up on complex migration work
   MIGRATION = 'migration'
+
+  # HIGHMEM: Jobs that process especially large amounts of data
+  # These are generally admin-facing reports that are rolled up into emails
+  # so they can take some time to process, but require the extra memory
+  # to ensure that they can complete without Heroku killing the dynos
+  HIGHMEM = 'highmem'
```

Comment:
Can you rename this to be `HIGH_MEMORY = high_memory`. That follows the pattern here of using full words (and which hopefully makes the code clearer).